internal/pkg/postgres/postgres.go

```diff
@@ -92,6 +92,50 @@ func (c *Client) CreateAlert(ctx context.Context, smeeAlert smee.Alert) (smee.Is
 		return smee.Issue{}, fmt.Errorf("unable to commit transaction: %w", err)
 	}
 
+	if err := c.unacknowledgeIssue(ctx, tx, issID); err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to unaknowledge issue: %w", err)
+	}
+
+	return smeeIss, nil
+}
+
+func (c *Client) AcknowledgeAlertsForIssue(ctx context.Context, issueID string) (smee.Issue, error) {
+	issID, err := strconv.Atoi(issueID)
+	if err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to parse issueID: %w", err)
+	}
+	tx, err := c.pool.Begin(ctx)
+	if err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to start transaction: %w", err)
+	}
+	defer func() {
+		_ = tx.Rollback(ctx)
+	}()
+
+	if err := c.acknowledgeAlertsforIssue(ctx, tx, issID); err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to acknowledge alerts from issue: %w", err)
+	}
+
+	unacknowledgedCount, err := c.unacknowledgedAlertCount(ctx, tx, issID)
+	if err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to get unacknowleged alerts from issue: %w", err)
+	}
+
+	if unacknowledgedCount == 0 {
```

Comment:
I'm confused about this logic. Wouldn't unacknowledged always be zero here since we just acknowledged all of the alerts?